test/test.js

```diff
@@ -8,7 +8,7 @@
  * Module dependencies.
  */
 var should 		= require('should');
-var OO 			= require('../index');
+var OO 			= require('../dist/index').default;
```

Comment:
That's happening when your're using `export default` in the ES6 syntax and compile it through babel and then trying to use the old syntax `require`. I don't love that mix, but I preferred to do the most changes in the src not in the test so we can be sure that the code didn't got broken.

We can't do a regular `require` from the source since it a ES6 syntax, the only choice we've for coverage is to use sourceMaps to reveal the ES6 source with correct coverage far away from the big compiled code